hook.php

```diff
@@ -57,7 +57,7 @@ function plugin_centreon_install($version)
                  ) ENGINE=InnoDB
                  DEFAULT CHARSET={$default_charset}
                  COLLATE={$default_collation}";
-        $DB->queryOrDie($query, $DB->error());
+        $DB->doQuery($query);
```

Comment:
```suggestion
        $DB->doQueryOrDie($query, __('Unable to create table ' . $table));
```

Reply:
`doQueryOrDie()` will be depreciated on GLPI 11